### PR TITLE
Add PHPCS to tools/

### PR DESCRIPTION
**Pull request by johnsaigle**
## Brief summary of changes

Adding PHPCS analysis to files in the `tools/` directory.

This will help the files stay more maintainable by cleaning them up and keeping them consistent with our rules elsewhere in the codebase.

There's more work to be done so I'll leave this in draft state until it's ready.

So far I've run PHPCBF on all the tools and am gradually fixing what PHPCBF can't do automatically.

### CODE REVIEW

**Comment by johnsaigle** [comment]
When reviewing, make sure to add `?w=1` to the URL to hide changes relating only to whitespace. This cuts down the lines added/deleted to about 1/3rd of those displayed by default.

**Comment by johnsaigle** [comment]
I've run into difficult issues with fixing some PHPCS errors within the tools.

This should be resolved by #5002. Most of the errors have to do with overly-long array declarations. This issue is described in depth in that PR.

As a result I'm adding Blocked to this PR.

File: tools/test.php [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?php
```
Comment (xlecours) [inline tools/test.php]: I am not sure if this file should be here.

File: test/run-php-linter.sh [hunk 2]
```diff
@@ -8,14 +8,13 @@ do
...
+# Run PHPCS on all .php and .inc files in folders: 
+# php/
+# htdocs/
+# modules/
+# The old API files under htdocs/ are ignored.
+vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php,inc --colors php/ htdocs/ modules/ --ignore=htdocs/api/*|| exit $?;
 
 vendor/bin/phpmd php/libraries text docs/LorisPHPMD.xml || exit $?;
+
+vendor/bin/phpcs --standard=docs/LorisCS.xml --extensions=php/php,inc/php tools/ || exit $?;
```
[the remark this answers is not in the document]
Comment (johnsaigle) [reply test/run-php-linter.sh]: It's done in a more robust way here: https://github.com/aces/Loris/pull/5002

**Comment by johnsaigle** [comment]
I'm going to reissue this instead of rebasing it.

**Pull request closed without merging**